.github/workflows/release-flux-artifact.yml

```
@@ -0,0 +1,95 @@
+name: Build Flux Artifact
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - flux/*/**
+  release:
+    types: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)